_data/simple-icons.json

```
@@ -13086,6 +13086,24 @@
             "hex": "258AAF",
             "source": "https://www.sitepoint.com"
         },
+        {
+            "title": "SiYuan",
+            "hex": "D23F31",
+            "source": "https://b3log.org/brand-marking.html",
```

Did you generate this from the PNG, or from the SVG in the site header? If the latter, we should change the source to reflect that.

```suggestion
            "source": "https://b3log.org/siyuan",
```

---

The original version of this icon was a bitmap image (`*.psd`), the SVG version is converted from bitmap. So I think `https://b3log.org/brand-marking.html` is a better than `https://b3log.org/siyuan`.

---

I can only find `.png` files on that page, am I missing something?

---

> I can only find `.png` files on that page, am I missing(失踪) something?

Oh, I'm one of the core contributors to this open source project, so I have a undisclosed draft of this icon.